security/pfSense-pkg-snort/files/usr/local/pkg/snort/snort_generate_conf.php

```diff
@@ -1456,7 +1456,7 @@
 
 // Iterate configured HTTP_INSPECT servers and write them to string if HTTP_INSPECT enabled
 if ($snortcfg['http_inspect'] <> "off") {
-	if (!is_array($snortcfg['http_inspect_engine']['item'])) {
+	if (!array_get_path($snortcfg, 'http_inspect_engine/item')) {
```

Comment:
As above, array_init_path() here can make this less verbose

Reply:
Implemented the suggestion.